country_by_country/img_table_extraction/llama_parse_extractor.py

```diff
@@ -40,13 +41,17 @@ def __init__(self, **kwargs: dict) -> None:
         You are free to define any parameter LlamaParse recognizes
         """
         self.kwargs = kwargs
+        self.type = "llama_parse"
 
         # Load LLAMA_CLOUD_API_KEY from .env file
         load_dotenv()
```

Comment:
Usually loading env vars takes place once in a well defined location in a code base.  A good place could be the country_by_country `__main__.py` file.

Reply:
I moved env loading to `country_by_countr/__main__.py` file